src/js/components/Select/index.d.ts

```diff
@@ -8,6 +8,15 @@ import {
   PlaceHolderType,
 } from '../../utils';
 
+type OptionType = string | number | { label: string; value: string | number };
+
+// Define the event type for the onChange function
+interface SelectOnChangeEvent {
+  option?: number; // Option selected from the dropdown
```

Comment:
I think some of these types are to narrow. 

For example if you now look at the CreatOption.tsx story we are getting an error on the `option` since it is not a number

Reply:
I also wonder if it would help to test this if we either change locally the stories we have to be typescript to ensure we are not breaking anything as well as the tests we can change to be typescript as well to test.

Reply:
Hi @britt6612 ,
Thanks for the point out,  Will look into this.

Reply:
@britt6612 
I have fixed the issues on CreateOptions.tsx, Please have a look.
Thanks